sklearn/ensemble/tests/test_base.py

```diff
@@ -24,8 +24,8 @@
 
 def test_base():
     # Check BaseEnsemble methods.
-    ensemble = BaggingClassifier(base_estimator=Perceptron(random_state=None),
-                                 n_estimators=3)
+    ensemble = BaggingClassifier(
+        base_estimator=Perceptron(tol=1e-1, random_state=None), n_estimators=3)
```

Comment:
I set it to avoid a `FutureWarning`, and I don't set it to `1e-3` to avoid a `ConvergenceWarning`.
It is quite hard to have a default `tol` that fits for every tests.